src/ui/templates/partials/confirm-email-page-content.njk

```diff
@@ -0,0 +1,34 @@
+{% if exporterEmail %}
+    <p data-cy="we-sent-link-to">{{ CONTENT_STRINGS.WE_SENT_LINK_TO }} {{ exporterEmail }}</p>
```

Comment:
indentation in this file seems to be off a little.  I think all indented when not in the if.  And line 3 should remove indentation

Reply:
Whoops, fixed